docs/source/paper.md

```diff
@@ -28,9 +28,13 @@ bibliography: paper.bib
+On the software level these problems are perhaps best addressed by using messaging brokers that take responsibility for guaranteeing the durability and atomicity of messages and often enable event-based communication.  Indeed, solutions such as [RabbitMQ](https://www.rabbitmq.com/) see widespread adoption in industry, however, adoption in academia has been more limited, with home-made queue data structures, race condition susceptible locks and polling based solutions being commonplace. This is likely due to message brokers typically having complex APIs (which reflect the non-trivial nature of the underlying protocol) as well as the lack of familiarity with event-based programming in general within the community.  [KiwiPy](https://kiwipy.readthedocs.io/en/latest/) was designed specifically to address both these issues, by providing a tool that allows building robust, event-based systems with an interface that is as simple as possible.
 
-KiwiPy provides three main message types to the user, task queues, Remote Procedure Calls (RPCs), and, broadcasts. These are all exposed via one class called the `Communicator` which can be trivially constructed by providing a URI string pointing to the RabbitMQ server. This starkly contrasts with the several steps that are required to establish a connection to perform each of the message types listed above using [standard libraries](https://www.rabbitmq.com/getstarted.html). Furthermore, by default, kiwiPy creates a separate communication thread that the user never sees, allowing them to interact with the communicator using familiar Python syntax, without the need to be familiar with either coroutines or multithreading.  This has the additional advantage that kiwiPy will maintain heartbeats (a periodic check to make sure the connection is still alive) with the server whilst the user code can be doing other things. Heartbeats are an essential part of RabbitMQ's fault tolerance whereby two missed checks will automatically trigger the message to be requeued to be picked up by another client.
+KiwiPy provides three main message types to the user: task queues, Remote Procedure Calls (RPCs), and, broadcasts.
+These are all exposed via one class called the `Communicator` which can be trivially constructed by providing a URI string pointing to the RabbitMQ server.
+This is in contrasts with the several steps that are required to establish a connection to perform each of the communication types listed above using [Pika](https://pika.readthedocs.io/en/stable/), the standard library used in RabbitMQ [tutorials](https://www.rabbitmq.com/getstarted.html).
```

Comment:
```suggestion
This is in contrast with the several steps that are required to establish a connection, to perform each of the communication types listed above, using [Pika](https://pika.readthedocs.io/en/stable/), the standard library used in RabbitMQ [tutorials](https://www.rabbitmq.com/getstarted.html).
```

Reply:
Changed this slightly to be more clear about what I meant.